testcases/crypto/rsa_func.c

```diff
@@ -1369,6 +1369,20 @@ CK_RV do_WrapUnwrapRSA(struct GENERATED_TEST_SUITE_INFO * tsuite)
             unwrap_tmpl_len = 2;
         }
 
+        switch (wrap_mech.mechanism) {
```

Comment:
well, above if involves keygen_mech.mechanism and wrap_mech.mechanism, but the switch only covers wrap_mech.mechanism. Certainly the code could be incorporated into the CKM_RSA_X_509 case in the switch, but not sure if this would really make it easier to read.